data/remnant missions.txt

```diff
@@ -493,5 +592,771 @@ mission "Remnant: Technology Available"
+				accept
+	
+	on complete
+		outfit "Catalytic Ramscoop" -2
+		payment 1000000
+		conversation
+			`You visit the local shipyard and eventually find the engineer busily repairing a damaged Starling. You inform her that you have brought a pair of "Catalytic Ramscoops" for her to study. She quickly tightens a couple more connectors and jumps down from her perch, gesturing at an assistant to transfer you the credits you were promised. "Thank you! Our maps of human space are old, and if they are still accurate, you traveled a very long way to bring these to us." Her chanting voice echoes off the starling above you with an interesting resonance.`
+			`	"Thank you for your help," she says. "If we can unlock the secrets of these scoops, it will give our ships greater range as they search for a safer refuge." As she picks up the remote control for the small loading truck with the two ramscoops she trills at you over her shoulder. "Oh, my name is Tali. I'll let you know what I find."`
+
+mission "Remnant: Plasma Cannons"
+	name "Retrieve Plasma Cannons for the Remnant"
+	description "An engineer on <planet>, a Remnant world, has offered to pay you one million credits in exchange for obtaining two Plasma Cannons."
+	minor
+	source Viminal
+	to offer
+		has "license: Remnant"
+		random < 50
+	on offer
+		require "Plasma Cannon"
+		conversation
+			`As you are browsing the spaceport shops, a man approaches you and introduces himself as a weapons specialist at one of the Remnant research facilities. He pauses hesitantly before beginning to chant: "As your ship is the only one we have seen in generations that has weapons from human space, we are intrigued by it. Could we take a closer look?"`
+			choice
+				`	"Yes, you are welcome to examine it."`
+					goto deep
+				`	"No, stay away from my ship."`
+					decline
+				`	"Yes, but don't touch anything."`
+
+			`	"Don't worry," he sings with a tinge of reproach. "I am a Remnant, not a Korath."`
+				goto more
+
+			label deep
+			`	He clips his harness to one of the overhead lifts and presses a button on the back of his gauntlet, and is promptly whisked up into the air. The lift appears designed to give engineers and techs almost as much freedom as a zero-g environment by suspending them with just enough force to counteract gravity, so they can move around a ship with ease. After a quick once-over he quickly gravitates to the weapon ports and turret hardpoints.`
+
+			label more
+			`	After examining <ship> for quite some time, he calls down to you. "I have never seen weapons designed to handle such a high heat output! Is this some kind of plasma based weapon?" he asks, gesturing at a Plasma Cannon. You nod, and he continues, "Intriguing. We have noticed that the Korath ships seem to run particularly hot, and have speculated that overheating them might be an effective means of disabling them. I would enjoy researching these a bit more. If you could acquire some for us you would be well compensated."`
```

Comment:
Again, an issue with the deep/not deep examination distinction; `he calls down to you` from where? I said don't touch anything. why the heck are you up on my ship?

Perhaps for this mission and the mission prior, remove one of the `yes` options and change the other option to something like `yes, but don't break it`, which leads into the `I'm a Remnant, not a Korath` line which leads to the current `deep` labels then down to `more`.

Reply:
Hmmmm. You're right, why *is* he up on your ship? 

Anyway, I don't want to force the "being overprotective of my ship" on the player by eliminating one of the choice options, so I split it up slightly differently and reworded it a bit to make it clear that with the one option he's up on the ship examining it, whereas with the second option he stays on the ground and doesn't touch anything. 

Also corrected the case of the plasma cannons and removed some unneeded quotation marks.